content/blog/2022-06-14-DVC-VS-Code-extension.md

```diff
@@ -0,0 +1,137 @@
+---
+title: >
+  Turn Visual Studio Code into an ML experimentation platform with the DVC
+  extension
+date: 2022-06-14
+description: >
+  Today we are releasing the DVC extension, which brings a full ML
+  experimentation platform to VS Code.
+descriptionLong: >
+  Today we are releasing the DVC extension, which brings a full ML
+  experimentation platform to VS Code. Manage your data, run experiments,
+  compare metrics, and visualize plots, all from the comfort of your IDE.
+picture: 2022-06-14/header.png
+author: rob_dewit
+# commentsUrl:
+tags:
+  - VS Code
+  - DVC
+  - MLOps
+  - Release
+---
+
+Since its beta release in 2017, DVC has become an essential tool for many data
+science teams. Its data versioning capabilities, reproducible pipelines, and
+experiment tracking features are at the core of our ecosystem of open MLOps
+tools.
+
+Today we are proud to launch a new product that extends how ML teams can use
+DVC: our extension for Visual Studio Code.
+
+With this extension you get a full VS Code-native experimentation platform for
+your machine learning projects. You can control datasets, run experiments, view
+metrics, create plots, and much more. All in one place in your IDE.
+
+[Install the DVC extension from the marketplace to get started.](https://marketplace.visualstudio.com/items?itemName=Iterative.dvc)
+
+# Why an extension?
+
+We built DVC to expand upon the Git workflow. This approach has made it a
+powerful tool that can easily be picked up by users familiar with Git. However,
+it can be more difficult for those less familiar with CLI tools to get
+accustomed to using DVC. Moreover, while DVC provides many useful
+visualizations, a CLI tool can only take things so far.
+
+Therefore we wanted to incorporate DVC's functionality in a graphical interface.
+[VS Code is the IDE of choice for many](https://insights.stackoverflow.com/survey/2021#section-most-popular-technologies-integrated-development-environment),
+including most of us here at Iterative. It seemed like the natural choice for a
+platform to make DVC more versatile, accessible, and useful.
+
+With this extension, we want to achieve the following:
+
+- Move the ML experimentation workflow into your IDE
```

Comment:
Not sure this captures why moving the experimentation workflow into your IDE is helpful. It can be both easier to use and more powerful since you can start an experiment with the click of a button, click to compare specific experiments, etc.